pyjobshop/cpoptimizer/ConstraintsManager.py

```diff
@@ -34,6 +34,7 @@ def _job_spans_tasks(self):
             job_var = self._job_vars[idx]
             job_task_vars = [self._task_vars[task] for task in job.tasks]
 
+            # TODO job_var is present iff at least one task is present
```

Comment:
This is a bit tricky. We currently don't have optional jobs, so CP Optimizer's span function will always select one optional tasks.

Reply:
For throughput we need optional jobs anyway.